osu.Game/Rulesets/Mods/DifficultyBindable.cs

```diff
@@ -18,7 +18,7 @@ public class DifficultyBindable : Bindable<float?>
         /// An internal numeric bindable to hold and propagate min/max/precision.
         /// The value of this bindable should not be set.
         /// </summary>
-        internal readonly BindableFloat CurrentNumber = new BindableFloat
+        internal readonly BindableFloat DisplayBindable = new BindableFloat
```

Comment:
I'm not sure this is named any better than it was previously. Maybe something like `CurrentConstraints` or `BindableConstraints` would work better. But this is all really quite out of scope.

Reply:
Now that I look at it again actually, perhaps `CurrentNumber` is fine either way. Note that I didn't use `Constraints` as the bindable gets fully used in the slider either way, so it doesn't make sense to bind a slider to a "constraints" bindable rather than a "current" or "display".

I'll revert the naming.